testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/admin/ClientTest.java

```diff
@@ -536,6 +539,8 @@ public void offlineUserSessions() throws IOException {
         List<UserSessionRepresentation> offlineUserSessions = realm.clients().get(id).getOfflineUserSessions(0, 100);
         assertEquals(1, offlineUserSessions.size());
         assertEquals("testuser", offlineUserSessions.get(0).getUsername());
+        org.hamcrest.MatcherAssert.assertThat(offlineUserSessions.get(0).getLastAccess(),
```

Comment:
Thanks for this. Is it possible to use a bit bigger time instead of 1000 (EG. 10000)? In some environments, testsuite is quite slow and I afraid that with 1000 ms, it can fail this condition due this slowness.

Reply:
@mposolda Thanks. I changed.